src/runtime_stack_check.js

```diff
@@ -14,11 +14,14 @@ function checkStackCookie() {
   if (HEAP32[0] !== 0x63736d65 /* 'emsc' */) throw 'Runtime error: The application has corrupted its heap memory area (address zero)!';
 }
 
+#if !MINIMAL_RUNTIME // MINIMAL_RUNTIME moves this to a JS library function
```

Comment:
can we just move it in general to a js library function? (in a separate PR preferably)

Reply:
I wanted to do that, but it is not possible in non-MINIMAL_RUNTIME because of create_runtime_funcs_asmjs() generates asm.js code for this, and the stack functionality was too hard to move altogether, due to preamble.js and postamble.js referring to it.